api/src/routes/verification/index.ts

```diff
@@ -41,7 +43,7 @@ export class VerificationRoutes {
 
 			const vc: VerifiableCredentialJson = await this.verificationService.verifyIdentity(
 				subject,
-				this.config.serverIdentityId,
+				this.keyResolver.resolve(this.config.serverIdentityId),
```

Comment:
in this class only the server identity id is needed. It would be better to pass this into the consrtuctor instead of the complete Keyresolver. This makes it easier to call and easier to test since you have less dependencies.

Imo keyResolve.resolve should be only called once the server starts instead of everytime it uses the serverIdentityId since it is in memory afterwards and if the var is not available it should crash on startup